community/kernel/src/main/java/org/neo4j/graphdb/impl/notification/NotificationCode.java

```diff
@@ -184,7 +184,12 @@
     START_UNAVAILABLE_FALLBACK(
             SeverityLevel.WARNING,
             Status.Statement.PlannerUnavailableWarning,
-            "START is unsupported for current CYPHER version, the query has been execute by an older CYPHER version"
+            "START is not supported for current CYPHER version, the query has been executed by an older CYPHER version"
```

Comment:
Is this still used or lying around for compatibility reasons? I thought this PR removed the fallback.